ghcide/src/Development/IDE/Plugin/CodeAction/ExactPrint.hs

```diff
@@ -50,22 +49,15 @@ data Rewrite where
 -- | Convert a 'Rewrite' into a 'WorkspaceEdit'.
 rewriteToEdit ::
   DynFlags ->
-  Uri ->
   Anns ->
   Rewrite ->
-  Either String WorkspaceEdit
-rewriteToEdit dflags uri anns (Rewrite dst f) = do
+  Either String [TextEdit]
+rewriteToEdit dflags anns (Rewrite dst f) = do
   (ast, (anns, _), _) <- runTransformT anns $ f dflags
-  let editMap =
-        HMap.fromList
-          [ ( uri,
-              List
-                [ TextEdit (fromJust $ srcSpanToRange dst) $
+  let editMap = [ TextEdit (fromJust $ srcSpanToRange dst) $
                     T.pack $ tail $ exactPrint ast anns
```

Comment:
If the above is the case, we also need to delete `tail` here, as it was intended to delete preceding spaces IIUC, right? (at least in #1264, this with `setEntryDPT` converts `import` into `mport`!)

Reply:
Yes, that's correct

Reply:
Will you make such changes also in this PR, or do you think it deserves another PR?

Reply:
I think it's better left for another PR